mis_builder/tests/test_mis_report_instance.py

```diff
@@ -501,11 +497,7 @@ def test_query_company_ids(self):
         assert self.report_instance.query_company_ids == self.report_instance.company_id
         # create a second company
         c1 = self.report_instance.company_id
-        c2 = self.env["res.company"].create(
-            dict(
-                name="company 2",
```

Comment:
Another example here where you should keep the trailing comma to reduce the diff (sorry to nitpick but it's important to me).